packages/push/Cargo.toml

```diff
@@ -4,8 +4,13 @@ version = { workspace = true }
 authors = { workspace = true }
 description = { workspace = true }
 documentation = { workspace = true }
+repository = { workspace = true }
 edition = { workspace = true }
 license = { workspace = true }
+# Per definition crates.io compilers includes interpreters
+categories = ["science", "algorithms", "compilers"]
+keywords = ["evolutionary-computation", "genetic-programming", "interpreter", "stack-based", "programming-language"]
```

Comment:
I would add ("Push" or "Push Language") and "PushGP" here.

Is "compilers" the right term? Or "iterpreters" like you used on line 12?

Reply:
as I mentioned above the categories should match the ones defined by crates.io, and crates.io explicitly states that the compilers category includes interpreters

Reply:
( there is no interpreters category )